stride/problem/geometry.py

```diff
@@ -283,6 +285,47 @@ def default(self, geometry_type, *args, **kwargs):
 
             self.add(index, self._transducers.get(0), _coordinates)
 
+    def from_fullwave(self, geom_path, **kwargs):
+        """
+        Populates geometry container based from a Fullwave geometry pgy file
+
+        Parameters
+        ----------
+        geom_path : str
+            Path to .pgy Fullwave geometry file
+        scale : float, optional
+            Value to each scale the location values in all dimensions. Useful for unit conversion.
+            To transform cell units from Fullwave to metric units in Stride, scale should be the
+            same value as the model grid-spacing. Default 1.
+        disp : tuple or float, optional
+            Amount to displace in each dimension [units Metres]. Applied after scale. Default (0., 0., 0.)
+        dropdims : tuple or int, optional
+            Coordinate dimensions of .pgy file to drop (count from 0). Default ()
+        swapaxes : bool, optional
+            Permutes Fullwave storing format (depth, cross-line, in-line) to stride format
+            (in-line, depth, cross-line). Default False.
+
```

Comment:
change these for `drop_dims` and `swap_axes`

Reply:
Amended